src/features/profile/profile-read-view/profile-tabs/dishes-tab/dish-score-display/song-score-display/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Badge, Box, Flex, Image, Text, useMediaQuery } from '@chakra-ui/react';
+import React from 'react';
+import { Score } from 'types/core';
+import { defaultPixel } from 'types/styled';
+import { getAssetUrl } from 'utils/assets';
+import { getColorByDifficulty } from 'utils/song-difficulty-colors';
+import { useLocation } from 'wouter';
+
+// DON'T DELETE THIS...
```

Comment:
This file looks potato 🥔 
But this is pre-work for a button that says "View details"
On click a modal with a list of songs and scores that made up the dish score